### PR TITLE
Update READMEs with new template for IntelPyTorch Inference/Training Optimizations

### PR DESCRIPTION
**Pull request by alexsin368**
# Existing Sample Changes
## Description

Update READMEs according to new template: https://github.com/oneapi-src/oneAPI-samples/wiki/AI-Getting-Started-and-Features-and-Functionality-README-Template

Fixes Issue# N/A

## External Dependencies

None

## Type of change

Just README update

## How Has This Been Tested?

Testing not needed for README update since only formatting changed.

### CODE REVIEW

File: AI-and-Analytics/Features-and-Functionality/IntelPyTorch_InferenceOptimizations_AMX_BF16_INT8/README.md [hunk 1]
```diff
@@ -1,158 +1,165 @@
...
-You will need to download and install the following toolkits, tools, and components to use the sample.
-
-- **Intel® AI Analytics Toolkit (AI Kit)**
-
-  You can get the AI Kit from [Intel® AI Analytics Toolkit](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-analytics-toolkit-download.html).
-
-### For Intel® DevCloud
-
-The necessary tools and components are already installed in the environment. You do not need to install additional components. See *[Intel® DevCloud for oneAPI](https://DevCloud.intel.com/oneapi/get_started/)* for information.
+| Software                | Intel® Extension for PyTorch*
```
Comment (Marina-Zubova) [inline AI-and-Analytics/Features-and-Functionality/IntelPyTorch_InferenceOptimizations_AMX_BF16_INT8/README.md:24]: in AI template, there is a note under the table
```suggestion
> **Note**: AI and Analytics samples are validated on AI Tools Offline Installer. For the full list of validated platforms refer to [Platform Validation](https://github.com/oneapi-src/oneAPI-samples/tree/master?tab=readme-ov-file#platform-validation).
```

**Review by Marina-Zubova — CHANGES_REQUESTED** [review]
Thank you for updating the READMEs, I left some comments

File: AI-and-Analytics/Features-and-Functionality/IntelPyTorch_TrainingOptimizations_AMX_BF16/README.md [hunk 2]
```diff
@@ -1,44 +1,172 @@
...
 
 ## Prerequisites
 
 | Optimized for           | Description
 |:---                     |:---
-| OS                      | Ubuntu* 18.04 or newer
+| OS                      | Ubuntu* 22.04 or newer
 | Hardware                | 4th Gen Intel® Xeon® Scalable Processors or newer
-| Software                | Intel® Extension for PyTorch (IPEX)
+| Software                | Intel® Extension for PyTorch* (IPEX)
```
Comment (Marina-Zubova) [inline AI-and-Analytics/Features-and-Functionality/IntelPyTorch_TrainingOptimizations_AMX_BF16/README.md:24]: in AI template, there is a not under the table
```suggestion
> **Note**: AI and Analytics samples are validated on AI Tools Offline Installer. For the full list of validated platforms refer to [Platform Validation](https://github.com/oneapi-src/oneAPI-samples/tree/master?tab=readme-ov-file#platform-validation).
```

**Review by Marina-Zubova — CHANGES_REQUESTED** [review]

File: AI-and-Analytics/Features-and-Functionality/IntelPyTorch_TrainingOptimizations_AMX_BF16/README.md [hunk 3]
```diff
@@ -1,44 +1,172 @@
...
+
+
+
+
+
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------
 
->**Note**: Training is not performed using INT8 since using a lower precision will train a model with fewer parameters, which is likely to underfit and not generalize well.
 
 The sample tutorial contains one Jupyter Notebook and a Python script. You can use either.
```
Comment (Marina-Zubova) [inline AI-and-Analytics/Features-and-Functionality/IntelPyTorch_TrainingOptimizations_AMX_BF16/README.md:172]:
```suggestion
*Other names and brands may be claimed as the property of others. [Trademarks](https://www.intel.com/content/www/us/en/legal/trademarks.html)
```

File: AI-and-Analytics/Features-and-Functionality/IntelPyTorch_TrainingOptimizations_AMX_BF16/README.md
[the remark this answers is not in the document]
Comment (alexsin368) [reply AI-and-Analytics/Features-and-Functionality/IntelPyTorch_TrainingOptimizations_AMX_BF16/README.md:1]: My mistake. Just fixed it.

**Review by Marina-Zubova — APPROVED** [review]

**Pull request merged**